internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/JobStatusQueryResponse.java

```diff
@@ -92,6 +108,20 @@ public void setExtraDetails(Map<String, String> extraDetails) {
     this.extraDetails = extraDetails;
   }
 
+  /**
+   * N.B.: Older versions of the controller did not support these timestamps, so this map can be null.
+   *
+   * @return a map of region name -> UNIX Epoch timestamp indicating when the value corresponding to the same key in
+   *         {@link #getExtraInfo()} was last updated, or null if not available.
+   */
+  public Map<String, Long> getExtraInfoUpdateTimestamp() {
+    return this.extraInfoUpdateTimestamp;
+  }
+
+  public void setExtraInfoUpdateTimestamp(Map<String, Long> extraInfoUpdateTimestamp) {
```

Comment:
Can you elaborate what kind of timestamp will be populated into this map?

Reply:
I assume some of the hooks would leverage this timestamp to decide the wait time after version swap, right?

Reply:
And this is for the aggregated results from child regions.

Reply:
Yeah, the hook could leverage these timestamps to decide if it waited long enough after a region got swapped.